### PR TITLE
Using Deep Learning #57 (Model-Simple feed-forward neural network)

### PR DESCRIPTION
**Pull request by ranamanish674zu**
I used deep learning (simple feed forward neural network) on the given datasest, but due to small size of dataset or something else model gone overfitted(training accuracy-100 %, testing accuracy-69%, Overfitting occurs when your model learns the training data too well, capturing noise and details that do not generalize to new, unseen data ) , so to resolve it i used Regularization(regularization techniques like L2 regularization (weight decay) to the model) , Dropout(Introduce dropout layers to the model to randomly set a fraction of input units to 0 at each update during training time, which helps prevent overfitting) , Early Stopping(Use early stopping to halt training when the validation loss stops improving) for balanced the training and testing accuracy. So after applying this i get training accuracy-above 80 % and testing accuracy-75%. after this i save the model into deeplearning.pkl file, if we developers want use this model then they can use by importing this file in the code. that's i solved issue #57 
image
image

### CODE REVIEW

**Comment by BamaCharanChhandogi** [comment]
Thank you for submitting your pull request! 🙌 We'll review it as soon as possible. In the meantime, please ensure that your changes align with our [CONTRIBUTING.md](https://github.com/BamaCharanChhandogi/Diabetes-Prediction/blob/main/CONTRIBUTING.md). If there are any specific instructions or feedback regarding your PR, we'll provide them here. Thanks again for your contribution! 😊

**Comment by BamaCharanChhandogi** [comment]
I will look at this PR later.

**Review by BamaCharanChhandogi — APPROVED** [review]
Can you create a separate file for deep learning, because we do not need deep learning as of now? I want to keep separate this.

**Review by BamaCharanChhandogi — CHANGES_REQUESTED** [review]
Can you create a separate file for deep learning, because we do not need deep learning as of now? I want to keep separate this
